src/assets/scripts/airport/Route/RouteModel.js

```diff
@@ -0,0 +1,133 @@
+import _uniqueId from 'lodash/uniqueId';
+
+/**
+ * Symbol that divides each route segment
+ *
+ * @property SEGMENT_SEPERATION_SYMBOL
+ * @type {string}
+ * @final
+ */
+const SEGMENT_SEPERATION_SYMBOL = '.';
+
+/**
+ * @class RouteModel
+ */
+export default class RouteModel {
```

Comment:
Yeah, that could be a thing. And yes, that is better verbage, particularly because `origin` and `destination` should eventually be used for the origin and destination airports. But yes, for a further example, the following is also acceptable:
```
Given that A-B-C-D-E-F-G-H are arranged sequentially in a circle shape, which is an airway called J123:

"A.J123.C..G..H.J123.E..F.J123.D..A..B..C..D.J123.H" is a valid route (albeit a squirrely one)

Disclaimer: You couldn't actually have a circular airway; it is just for example purposes
```

Any combination of things are fine; all you can't do is single-link with non-procedures (eg `A.B.C.D.E` or double-link procedures `A..J123.E`.